packages/auth/src/features/verifyCode/service.ts

```diff
@@ -74,8 +74,9 @@ export async function sendVerificationCode(
       Authorization: `Bearer ${await createToken(
         'auth',
         ['service'],
-        ['opencrvs:notification-user'],
-        JWT_ISSUER
+        ['opencrvs:notification-user', 'opencrvs:countryconfig-user'],
+        JWT_ISSUER,
+        true
```

Comment:
This boolean means that the token is for a system. That's not the case here no?

Reply:
Ah okay so it makes the token temporary. The name is confusing tbh

Reply:
Its not the case, no, but if we didnt have this, then this JWT is essentially valid for a month.  Theres a small risk of abuse of the SMS gateway.